scripts/catch/1.9.6/.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: generic
+
+sudo: false
+
+matrix:
+  include:
+    - os: osx
+      compiler: clang
+      osx_image: xcode8.2
+    - os: linux
+      compiler: clang
+      addons:
+        apt:
+          sources: [ 'ubuntu-toolchain-r-test' ]
+          packages: [ 'libstdc++-5-dev' ]
+
```

Comment:
Because header-only packages are platform agnostic, this needs only one matrix entry. (otherwise it will double-publish to the same location, which is harmless but not ideal). See https://github.com/mapbox/mason/blob/master/scripts/geometry/0.9.2/.travis.yml for how this should look.